service/src/test/resources/ehr/mapper/ehrExtract/ehrExtract.xml

```diff
@@ -0,0 +1,8160 @@
+<EhrExtract classCode="EXTRACT" moodCode="EVN">
```

Comment:
Shall we try and simplify this XML snippet?

Reply:
Have removed any components which were not required for the process to complete, whilst leaving in 2 components to ensure that the test is effective